### DISCUSSION TITLE
Insert -> Route not defined?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [awcodes/filament-curator#320] [Q&A]
When i click on "Insert" an image, i get error:

"Route [filament.admin.resources.media.edit] not defined."

**Comment by awcodes** [comment]
Are you using the plugin outside of the Panels package? i.e. with stand alone forms. If so, this is a bug. Feel free to file an issue if that is the case.

**Comment by hfallahnezhad** [comment]
If you are using a custom resource for curator, in addition to your panel provider plugins that you introduce your custom media resource, you should also set 'resource' => \App\Filament\Resources\YourCustomResource::class, in config/curator.php
